packages/sx05re/emuelec/bin/emuelec-utils

```diff
@@ -6,6 +6,47 @@
 # Source predefined functions and variables
 . /etc/profile
 
+function killall_midi()
+{
+	pkill -9 timidity
+  pkill -9 mt32d
+  pkill -9 fluidsynth
+	
+}
+
+function set_midi_source()
+{
+  local midi_source=$1
+  local midi_emulator=$2
+
+  killall_midi
+
+  [[ "${midi_emulator}" == "retrorun" ]] && return
+
+  local midi_output=OFF
+  
+  case ${midi_source} in
+    fluidsynth)
+      fluidsynth -is /storage/roms/bios/sc/SC-55.sf2 &
+      sleep 3
+      =$(cat /proc/asound/seq/clients | grep "Port   0 : \"S" | cut -d'"' -f2)
```

Comment:
it must be midi_output=$(cat /proc/asound/seq/clients | grep "Port   0 : "S" | cut -d'"' -f2) it lacks the midi_ouput var